lib/Driver/Compilation.cpp

```diff
@@ -120,7 +126,8 @@ Compilation::Compilation(DiagnosticEngine &Diags,
 static bool writeFilelistIfNecessary(const Job *job, DiagnosticEngine &diags);
 
 using CommandSet = llvm::SmallPtrSet<const Job *, 16>;
-
+using CommandSetVec = llvm::SetVector<const Job*>;
```

Comment:
Nitpick: I don't think the "Vec" abbreviation is buying anything here. "Vector" is fine.

Reply:
ok